.github/workflows/test-cli-with-database.yml

```diff
@@ -1,82 +1,82 @@
 name: CLI Pipeline
 on:
- pull_request:
+  pull_request:
 
 jobs:
- test-cli-with-database:
-   env:
-     NODE_ENV: CI
-     REDIS_URL: redis://localhost:6379
-     DATABASE_URL: "postgres://postgres:postgres@localhost/cli-test"
-     POSTGRES_URL: "postgres://postgres:postgres@localhost/cli-test"
-   services:
-     redis:
-       image: redis
-       options: >-
-         --health-cmd "redis-cli ping"
-         --health-interval 10s
-         --health-timeout 5s
-         --health-retries 5
-       ports:
-         - 6379:6379
+  test-cli-with-database:
```

Comment:
We really need to automate formatting for all files and not depend on people's editor for it. It's fine for now though :D

Reply:
yes we decided to post pon this to later but it is a bit annoying, cause mine also automatically clean up files ahah